content/api_chef_server_14.md

```diff
@@ -0,0 +1,8437 @@
++++
+title = "Chef Infra Server 14 API"
+draft = false
+
+aliases = ["/api_chef_server_14.html"]
+
+[menu]
+  [menu.infra]
+    title = "Chef Infra Server 14 API"
+    identifier = "chef_infra/managing_chef_infra_server/api_chef_server.md Chef Infra Server 14 API"
+    parent = "chef_infra/managing_chef_infra_server"
+    weight = 200
++++
+
+[\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/api_chef_server_14.md)
+
+The Chef Infra Server API is a REST API that provides access to objects
+on the Chef Infra Server, including nodes, environments, roles, users, organizations,
+cookbooks (and cookbook versions), and is used to manage an API client list and
+the associated RSA public key-pairs.
+
+## Requirements
+
+The Chef Infra Server API has the following requirements:
+
+-   The `Accept` header must be set to `application/json`.
+-   For `PUT` and `POST` requests, the `Content-Type` header must be set
+    to `application/json`.
+-   The `X-Chef-Version` header must be set to the version of the Chef
+    Infra Server API that is being used.
+-   A request must be signed by adding authentication headers.
```

Comment:
The quotes in 'Mixlib::Authentication` need to match.

Reply:
Is this comment in the right place?